### PR TITLE
Add Iterable.once to ccompat

### PR DESCRIPTION
**Pull request by mdedetrich**
See https://github.com/apache/incubator-pekko-connectors/pull/146 for context

### CODE REVIEW

**Comment by pjfanning** [comment]
There are probably other useful cases for this - places where we can remove Seq(t) and use this instead.

**Review by He-Pin — APPROVED** [review]
lgtm

File: actor/src/main/scala-2.12/org/apache/pekko/util/ccompat/package.scala [hunk 1]
```diff
@@ -114,3 +118,29 @@ class MapViewExtensionMethods[K, V, C <: scala.collection.Map[K, V]](
   def filterKeys(p: K => Boolean): IterableView[(K, V), C] =
     self.filter { case (k, _) => p(k) }
 }
+
+class IterableExtensionMethods[A, C <: scala.collection.Iterable[A]](
+    private val self: IterableView[A, C]) extends AnyVal {
```
Comment (He-Pin) [inline actor/src/main/scala-2.12/org/apache/pekko/util/ccompat/package.scala:123]: may need some fmt?

File: actor/src/main/scala-2.12/org/apache/pekko/util/ccompat/package.scala [hunk 2]
```diff
@@ -77,6 +77,30 @@ package object ccompat {
       build(i.TreeMap.newBuilder[K, V], source)
   }
 
+  private[pekko] implicit class IterableExtensions(private val fact: Iterable.type) extends AnyVal {
```
[the remark this answers is not in the document]
Comment (pjfanning) [reply actor/src/main/scala-2.12/org/apache/pekko/util/ccompat/package.scala:80]: If the code comes from scala-collection-compat, then our existing license has this covered. If it comes from Scala 2.12 library jar, then we probably need to update our license.

[the remark this answers is not in the document]
Comment (mdedetrich) [reply actor/src/main/scala-2.12/org/apache/pekko/util/ccompat/package.scala:80]: So its extremely loosely based on Scala 2.13 (it doesn't exist in Scala 2.12) due to differences between Scala 2.12 and 2.13

**Pull request merged**